src/components/side_panel/chart/combo_chart/combo_chart_config_panel.ts

```diff
@@ -0,0 +1,32 @@
+import { _t } from "../../../../translation";
+import { LineBarPieConfigPanel } from "../line_bar_pie_panel/config_panel";
+
+export class ComboConfigPanel extends LineBarPieConfigPanel {
+  static template = "o-spreadsheet-ComboConfigPanel";
+
+  get stackedLabel(): string {
+    return _t("Stacked combochart");
+  }
+
+  get shouldUseRightAxis(): string {
+    return _t("Use right axis for line series");
+  }
+
+  onUpdateStacked(stacked: boolean) {
+    this.props.updateChart(this.props.figureId, {
```

Comment:
omega nitpick: this could be prettified to one line

Reply:
In fact ti should be removed, as, as you said, stacked combo chart is a non-sense :)